src/dags/sql/stroomstoringen.py

```diff
@@ -0,0 +1,71 @@
+# The data contains epoch times.
+# In order to translate it to a timestamp, the data is updated.
+CONVERT_DATE_TIME = """
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_gemeld TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_schatting TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_eind TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN date_end TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_change TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_service_update TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN log_date TYPE varchar(50);
+
+UPDATE {{ params.tablename }}
+SET storing_datum_gemeld = to_timestamp(rpad(storing_datum_gemeld, 13, '0')::bigint / 1000),
+    storing_datum_schatting = to_timestamp(rpad(storing_datum_schatting, 13, '0')::bigint / 1000),
+    storing_datum_eind = to_timestamp(rpad(storing_datum_eind, 13, '0')::bigint / 1000),
+    date_end = to_timestamp(rpad(date_end, 13, '0')::bigint / 1000),
+    storing_datum_change = to_timestamp(rpad(storing_datum_change, 13, '0')::bigint / 1000),
+    storing_service_update = to_timestamp(rpad(storing_service_update, 13, '0')::bigint / 1000),
+    log_date = to_timestamp(rpad(log_date, 13, '0')::bigint / 1000)
+WHERE 1=1;
+COMMIT;
+
+ALTER TABLE {{ params.tablename }} ADD COLUMN storing_datum_gemeld_tmp TIMESTAMP
+    without time zone NULL;
+UPDATE {{ params.tablename }} SET storing_datum_gemeld_tmp = storing_datum_gemeld::TIMESTAMP;
```

Comment:
If you allow these columns to have a timezone you don't need an explicit cast to `::TIMEZONE`

Reply:
:+1: